src/event_loop.rs

```diff
@@ -45,11 +45,9 @@ pub struct EventLoop<T: 'static> {
 /// Target that associates windows with an [`EventLoop`].
 ///
 /// This type exists to allow you to create new windows while Winit executes
-/// your callback. [`EventLoop`] will coerce into this type (`impl<T> Deref for
-/// EventLoop<T>`), so functions that take this as a parameter can also take
-/// `&EventLoop`.
-pub struct EventLoopWindowTarget {
-    pub(crate) p: platform_impl::EventLoopWindowTarget,
+/// your callback.
+pub struct ActiveEventLoop {
```

Comment:
There was a reason that I included a lifetime here in https://github.com/rust-windowing/winit/pull/3299; this was done so that it'd be easier to change to a mutable reference in the future.

But I'm fine with not doing it yet, and maybe we should just be transitioning to a mutable reference directly.

Reply:
I mean the 0.31 will just take `&mut` everywhere, but from the user side they won't need much to change, I think. Also, I can't really do lifetime unless we agree to wrap window target into some struct carrying `&'a mut EventLoopWindowTarget`.